packages/typespec-ts/test/modularUnit/scenarios/models/serialization/additionalProperties.md

```diff
@@ -51,7 +51,7 @@ export function simpleModelSerializer(item: SimpleModel): any {
 export type _SimpleModelAdditionalProperty = number | string;
 
 export function _simpleModelAdditionalPropertySerializer(
-  item: _SimpleModelAdditionalProperty
+  item: _SimpleModelAdditionalProperty,
```

Comment:
pls revert this change.

Reply:
@v-jiaodi Could you priotize this fix? https://github.com/Azure/autorest.typescript/issues/3214

Reply:
pr : https://github.com/Azure/autorest.typescript/pull/3245